users/models.py

```diff
@@ -1,3 +1,52 @@
 from django.db import models
+from django.contrib.auth.models import AbstractUser
 
 # Create your models here.
+# Mentor model
+
+Semesters = (
+    ("1" , "1"),
+    ("2" , "2"),
+    ("3" , "3"),
+    ("4" , "4"),
+    ("5" , "5"),
+    ("6" , "6"),
+    ("7" , "7"),
+    ("8" , "8"),
+)
+Branches = (
+    ("1" , "CSE"),
+    ("2" , "IT"),
+    ("3" , "ECE"),
+    ("4" , "ELEC"),
+    ("5" , "MECH"),
+    ("6" , "CHEM"),
+    ("7" , "CIVIL"),
+    ("8" , "META"),
+    ("9" , "MIN"),
+    ("10" , "MCA"),
+    ("11" , "BIOMED"),
+    ("12" , "BIOTECH"),
+    ("13" , "MCA"),
+)
+
+class Mentor(AbstractUser):
+    email = models.EmailField(max_length=200)
+    name = models.CharField(max_length=200, null=False)
+    username = models.CharField(max_length=200,null=False,unique=True)
+    password = models.CharField(max_length=200)
+    alloted_team_ids = models.BigIntegerField()
+    branch = models.CharField(max_length=100,choices=Branches)
+    semester = models.CharField(max_length=100,choices=Semesters)
```

Comment:
Set this as `IntegerField` with `max_integer` as 8

Reply:
sir there is no property as max_integer in IntegerField

Reply:
https://stackoverflow.com/a/12026867

Reply:
done